.github/workflows/ci.yml

```diff
@@ -1,6 +1,20 @@
 ---
 name: ci/cd
 
+permissions:
+  actions: read
+  checks: read
+  contents: read
+  deployments: read
+  discussions: read
+  id-token: write
```

Comment:
[note] Explicitly enabling `id-token` is required to use OIDC.
https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

Reply:
P.S. Perhaps it is now enabled by default?
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Fixed to set minimum privilege 98cebd24efc994884da6f3934be8121e36c027bc